src/main/scala/rps/Game.scala

```diff
@@ -1,41 +1,31 @@
 package rps
-import java.util.Scanner
-import java.util.InputMismatchException
+import scala.io.StdIn.readLine
 
 object Game {
-  def moveRepr(move: Int) : String = {
+  val randomGen = scala.util.Random
+
+  def moveRepr(move: Int): String = {
     move match {
-      case 0 => return "Rock"
-      case 1 => return "Paper"
-      case 2 => return "Scissors"
-      case _ => return "Invalid move!"
+      case 0 => "Rock"
+      case 1 => "Paper"
+      case 2 => "Scissors"
+      case _ => "Invalid move!"
     }
   }
 
   def detectGameEndResult(playerMove: Int, computerMove: Int) : String = {
     (playerMove, computerMove) match {
-      case (x, y) if x == y => return "Draw!"
-      case (0, 1) => return "I Won!"
-      case (0, 2) => return "You Won!"
-      case (1, 0) => return "You won!"
-      case (1, 2) => return "I won!"
-      case (2, 0) => return "I won!"
-      case (2, 1) => return "You won!"
-      case (_, _) => return "Invalid Round!!"
+      case (x, y) if x == y => "Draw!"
+      case (0, 1) | (1, 2) | (2, 0) => "I Won!"
+      case (0, 2) | (1, 0) | (2, 1) => "You Won!"
+      case (_, _) => "Invalid Round!!"
     }
   }
 
   def play() : Unit = {
-    var scanner = new Scanner(System.in)
-    val r = scala.util.Random
-    print("Choose your move between Rock(0), Paper(1) or Scissors(2): ")
+    val myPlay = randomGen.nextInt(3)
+    val userPlay = readLine("Choose your move between Rock(0), Paper(1) or Scissors(2): ").toInt
```

Comment:
`.toInt` will throw an exception if I pass an invalid `Int`. This is ok for now, we'll handle this properly in future steps ;)